qdrant-landing/content/blog/what-is-a-vector-database.md

```diff
@@ -0,0 +1,222 @@
+
+The **creation** of vector data (so we can store this high-dimensional data on our vector database) is primarily done through **embeddings**.
+
+![](/blog/what-is-a-vector-database/Vector-Data.jpg)
+
+### How do Embeddings Work?
+
+Embeddings translate this high-dimensional data into a more manageable, **lower-dimensional** vector form that's more suitable for machine learning and data processing applications, typically through **neural network models**.
+
+In creating dimensions for text, for example, the process involves analyzing the text to capture its linguistic elements. 
+
+Transformer-based neural networks like **BERT** (Bidirectional Encoder Representations from Transformers) and **GPT** (Generative Pre-trained Transformer), are widely used for creating text embeddings. 
+
+Each layer extracts different levels of features, such as context, semantics, and syntax.
+
+![](/blog/what-is-a-vector-database/How-Do-Embeddings-Work_.jpg)
+
+
+The final layers of the network condense this information into a vector that is a compact, lower-dimensional representation of the image but still retains the essential information.
+
+
+![](/blog/what-is-a-vector-database/i-love-similarity.png)
+
+
+
+## Core Functionalities of Vector Databases
+
+
+### What is Indexing?
+
+Have you ever tried to find a specific face in a massive crowd photo? Well, vector databases face a similar challenge when dealing with tons of high-dimensional vectors. 
+
+Now, imagine dividing the crowd into smaller groups based on hair color, then eye color, then clothing style. Each layer gets you closer to who you’re looking for. Vector databases use similar **multi-layered** structures called indexes to organize vectors based on their "likeness." 
+
+This way, finding similar images becomes a quick hop across related groups, instead of scanning every picture one by one.
+
+
+![](/blog/what-is-a-vector-database/Indexing.jpg)
+
+
+Different indexing methods exist, each with its strengths. [HNSW](https://qdrant.tech/articles/filtrable-hnsw/) balances speed and accuracy like a well-connected network of shortcuts in the crowd. Others, like IVF or Product Quantization, focus on specific tasks or memory efficiency.
+
+
+#### What is Binary Quantization?
+
+Quantization is a technique used for reducing the total size of the database. It works by compressing vectors into a more compact representation at the cost of accuracy.
+
+[Binary Quantization](https://qdrant.tech/articles/binary-quantization/) is a fast indexing and data compression method available in Qdrant that allows for rapid vector comparison, which can speed up query processing times dramatically (up to 40x faster!).
+
+Think of each data point as a ruler. Binary quantization splits this ruler in half at a certain point, marking everything above as "1" and everything below as "0". This binarization process results in a string of bits, representing the original vector.
```

Comment:
As a reference for newer users, I wanted to find a link... this is the best I could do, you're welcome to substitute an alternative:

```suggestion
Think of each data point as a ruler. Binary quantization splits this ruler in half at a certain point, marking everything above as "1" and everything below as "0". This [binarization](https://deepai.org/machine-learning-glossary-and-terms/binarization) process results in a string of bits, representing the original vector.
```